packages/flutter_adaptive_scaffold/test/adaptive_scaffold_test.dart

```diff
@@ -43,11 +45,12 @@ void main() {
     expect(primaryNav, findsOneWidget);
 
     expect(tester.getTopLeft(body), const Offset(88, 0));
-    expect(tester.getTopLeft(sBody), const Offset(450, 0));
+    expect(tester.getTopLeft(sBody), const Offset(400, 0));
```

Comment:
Similar comment to https://github.com/flutter/packages/pull/2668/files#r985265588 here...